### PR TITLE
add rustls support for connect

### PR DESCRIPTION
**Pull request by jeizsm**

### CODE REVIEW

**Comment by fafhrd91** [comment]
Could you add test and entry to changes.md

**Comment by fafhrd91** [comment]
what is the status?

**Comment by jeizsm** [comment]
@fafhrd91 done

File: CHANGES.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 * Use new `Service<Request>` trait
 
+* Add `rustls` support
```
Comment (fafhrd91) [inline CHANGES.md]: could you move this line to `actix-connect/CHANGES.md`

**Comment by fafhrd91** [comment]
could you rename tests https://travis-ci.org/actix/actix-net/jobs/562952308#L783

**Comment by fafhrd91** [comment]
thanks!

**Pull request merged**